doc/sphinx/proof-engine/tactics.rst

```diff
@@ -4811,3 +4811,83 @@ references to automatically generated names.
    :name: Mangle Names Prefix
 
    Specifies the prefix to use when generating names.
+
+Performance-oriented tactic variants
+------------------------------------
+
+.. tacn:: change_no_check @term
+   :name: change_no_check
+
+   For advanced usage. Similar to :n:`change @term`, but as an optimization,
+   it skips checking that :n:`@term` is convertible to the goal.
```

Comment:
Minor grammar nit (not strictly necessary, but I think it reads better):
```suggestion
   it skips checking that :n:`@term` is convertible with the goal.
```

Reply:
Your call, but I wouldn't. The manual prefers "A and B are convertible", followed by "convertible to", then "convertible with".